README.md

```diff
@@ -28,21 +30,41 @@ The library can be imported as a maven dependency:
 
 ## Usage
 
+### Prerequisites
+
+Before your service will be able to send an email, you'll need to define a HTML and TXT template, and add it to the [`assets/email-templates`](https://github.com/companieshouse/chs-notification-api/tree/develop/assets/email-templates) directory of the [CHS Notification API](https://github.com/companieshouse/chs-notification-api). Once the templates are written, register them with the Notification API by adding an entry to [`assets/template_registry.yaml`](https://github.com/companieshouse/chs-notification-api/blob/develop/assets/template_registry.yaml). For example:
+
+```yaml
+'suppression_application_received':
+	'html-template': "suppression_application_received"
+	'sent-by-app-ids': [ 'suppression-api' ]
```

Comment:
this red filling is a bit weird, can we remove it

Reply:
It's a changefile artefact, representing a Tab. It isn't actually there.

Reply:
However, swapping the tabs for spaces seems to have resolved it.